examples/ensemble/plot_forest_importances.py

```diff
@@ -11,6 +11,9 @@
 As expected, the plot suggests that 3 features are informative, while the
 remaining are not.
 
+For a more extended example see
```

Comment:
This change does not make sense: you are linking the example in the example itself. You might want to review the instruction of the original issue.